index.js

```diff
@@ -240,7 +240,7 @@ const pkg = getPackageJson();
       );
     }
 
-    const remoteRepo = `https://${process.env.GITHUB_ACTOR}:${process.env.GITHUB_TOKEN}@github.com/${process.env.GITHUB_REPOSITORY}.git`;
+    const remoteRepo = `https://${process.env.GITHUB_ACTOR}:${process.env.GITHUB_TOKEN}@${process.env.CUSTOM_GIT_URL || "github.com"}/${process.env.GITHUB_REPOSITORY}.git`;
```

Comment:
how do you set this ENV in your action file?

I'd prefer `process.env['INPUT_CUSTOM-GIT-URL']` with `custom-git-url `listed under `inputs` https://github.com/phips28/gh-action-bump-version/blob/master/action.yml#L9C1-L9C8

Reply:
I've added the proposed changes